python/lsst/cp/pipe/defects.py

```diff
@@ -529,6 +539,48 @@ def findHotAndColdPixels(self, exp, imageType, setMask=False):
             footprintList += mergedSet.getFootprints()
 
         defects = measAlg.Defects.fromFootprintList(footprintList)
+
+        # If there's a column with on and off defects, mask all the pixels in between.
+        # Get the x, y values of the defects.
+        x, y = [], []
+        for defect in defects:
+            bbox = defect.getBBox()
+            x.append(bbox.getMinX())
+            y.append(bbox.getMinY())
+        x = np.array(x)
```

Comment:
I imagine that this is the kind of thing where the amount of developer time is >> the wall-time that will ever be saved when implementing, likely by >3 orders of magnitude, if not 6. You're not wrong, but I just don't think it will actually matter in reality. If I'm missing something about why this might be important though, please do let me know, I might just not have thought about it correctly.

Reply:
The code is also not what you want. Getting a list of lower left coordinates of all the bounding boxes does not tell you anything about the pixels that are masked. What you need is to get the mask from the bounding box (and `Defects` has a method for that).

Reply:
I'm confused. Even in the case that the defect has only one pixel width it is true that the bounding boxes do not tell us anything about the pixels that are masked? The code was written to handle bad pixels in columns.  What do they bounding boxes us about? In any case, I will look for the mask of the bounding box that you talk about and see if I can use that information in this case. Thanks.